lib/tasks/daily_vote_processing.rake

```diff
@@ -283,5 +283,26 @@ namespace :populate_database do
 
     @round.process_round
     @round2.process_round
+
+    @private_game1 = Game.create!(length_in_days: 151, guess_lead_time: 3, player_cap: 315, game_type: 0, results: nil)
```

Comment:
This needs to be game type 1

Reply:
Thank you for than clarification. I have this updated and will push up once I get the other updates cleared.